README.md

```diff
@@ -298,12 +318,38 @@ Finally, another distinguishing feature of Marko is that it supports _asynchrono
 
 # Usage
 
+## Template Loading
+
+Marko provides a custom Node.js require extension that allows Marko templates to be required just like a standard JavaScript module. For example:
+
+```javascript
+// The following line installs the Node.js require extension
+// for `.marko` files. Once installed, `*.marko` files can be
+// required just like any other JavaScript modules.
+require('marko/node-require').install();
+
+// ...
+
+// Load a Marko template by requiring a .marko file:
+var template = require('./template.marko');
+```
+
+_NOTE: The require extension only needs to be installed once, but it does not hurt if it is installed multiple times. For example, it is okay if the main app registers the require extension and an installed module also registers the require extension. The require extension will always resolve the proper `marko` module relative to the template being required so that there can still be multiple versions of marko in use for a single app._
+
+However, if you prefer to not rely on the require extension for Marko templates, the following code will work as well:
+
+```javascript
+var template = require('marko').load(require.resolve('./template.marko'));
+```
+
+A loaded Marko template has multiple methods for rendering the template as described in the next section.
+
 ## Template Rendering
 
 ### Callback API
 
 ```javascript
-var template = require('marko').load('template.marko');
+var template = require('./template.marko');
```

Comment:
In the latest version of marko the only cache is the Node.js require cache (i.e. `require.cache`). In order to reload a template without restarting the server it is necessary to delete the entry out of the require cache. This is what the hot-reload module does (see https://github.com/patrick-steele-idem/marko/blob/7c00085380ea318a9b9a935760e5e323d12612b7/hot-reload/index.js#L92-L93)